### DISCUSSION TITLE
Filters in convolutional blocks

### DISCUSSION THREAD

**Discussion by janiceylau** [talmolab/sleap#1252] [General]
Hello! I wonder what "filters" in convolutional blocks (or in "Model" at "run training" at the sleap GUI) actually mean, as I am doing some hyperparameter tuning. Does it mean the feature channels (e.g. the ones in UNet whose numbers double when going down the downsampling path) instead of the kernels? Does "filter rate" mean the rate they increase, i.e. double if "filter rate" = 2? I got this idea from your paper. Just to make sure I do not misunderstand. Thanks very much!

**Comment by janiceylau** [comment]
Oh I guess it also refers to the kernels. I misunderstood it as another feature of kernels that did not make sense to me.

**Reply by roomrys** [reply]
Hi @janiceylau,

Yes, the filters refer to kernels! Increasing the number of filters increases the level of detail captured at each level in the unet, but also increases computational cost. Just as you inferred, filter rate is the rate at which the number of filters change across downsampling/encoding blocks. 

Super cool that you are reading through the paper! Let us know if you have more questions.

Thanks,
Liezl

**Reply by janiceylau** [reply]
Hi Liezl, 

Nice! Thanks for your reply! :)

Janice
